ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -379,8 +391,101 @@ public class BottomSheetController: UIViewController {
         constraints.append(contentsOf: makeLayoutGuideConstraints())
 
         NSLayoutConstraint.activate(constraints)
+
+        updateAppearance()
+    }
+
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        tokenSet.update(fluentTheme)
```

Comment:
(Trying to get a hang of the tokenization process) - why do we do this update in viewDidAppear?

Reply:
This is how we update the token set if the theme changed and it's usually done in didMoveToWindow, but since BottomSheet is a UIViewController (rather than a UIView), this is the closest equivalent method. Ideally, the theme could be changed immediately (in the case the view is already showing), but I don't think that's possible currently (there's this TODO commented in DemoAppearanceView.swift):
<img width="398" alt="image" src="https://user-images.githubusercontent.com/31874971/212220510-3364a38a-75b2-4f68-b343-373aab925410.png">

Reply:
I seee. Thanks for the explanation!